tools/IndexStore/CMakeLists.txt

```diff
@@ -12,17 +12,17 @@ set(LIBS
   clangIndexDataStore
 )
 
-set(LLVM_EXPORTED_SYMBOL_FILE ${CMAKE_CURRENT_SOURCE_DIR}/IndexStore.exports)
-
-set(ENABLE_SHARED SHARED)
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Windows)
+  set(LLVM_EXPORTED_SYMBOL_FILE ${CMAKE_CURRENT_SOURCE_DIR}/IndexStore.exports)
+endif()
```

Comment:
I know I mentioned that I'd be fine with not using the export list for windows, but it was not clear to me why you wanted to do that ? I mean what is the benefit with *not* setting the exports list for windows ?

Reply:
Because when you build without `-fblocks` you need to remove the methods guarded by `__feature(blocks)`.  I don't think that it is worth the extra work of trying to do something to merge multiple export lists (and we do not have a way to conditionalize the list).  We *could* duplicate the list and have two copies but they can get out of sync, so that is less than ideal.  Removing it on windows and relying on the dllstorage attributes seems the best approach.

Reply:
Ok, thanks for the explanation!